common/helm-repositories/kubecost.yaml

```
@@ -0,0 +1,10 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
```

Can we use the latest api?

```suggestion
apiVersion: source.toolkit.fluxcd.io/v1
```

---

maybe we can but the rest of the helmrepository resources use v1beta2 so i ll just keep it as is for now to avoid any unexpected issues (like i am not sure if the version agnostic cli or the chart museum parser or the tooling that downloads helm charts etc., will be affected. most likely not, but i am being extra cautious here). we can update all helmrepos at once in future.